lib/Optimizer/GraphOptimizer/ConstantFolding.cpp

```diff
@@ -311,7 +312,10 @@ bool glow::ConstantFold::run(Function *F, const CompilationContext &cctx) {
     }
 
     // Compute the constant value of the node.
-    std::vector<Constant *> constResults = constantFoldNodeImpl(*backend, N);
+    std::vector<Constant *> constResults;
```

Comment:
Maybe use a SmallVector here to avoid heap allocations?

Reply:
We'd have to replace the vector everywhere in this file, but we could do it. I'm not sure theres a sensible number of results to use for the SmallVector though, it scales proportionally with the size of the network.

Reply:
Yeah, let's keep it simple. Ignore my previous comment!